konveyor/pelorus
# Make exporters log version information on startup

## Describe the behavior changes introduced in this PR
Logs version information at startup, obtained from environment variables set by a source to image build.

## Linked Issues?

resolves #379

## Testing Instructions

1. Deploy an exporter from `source_url: https://github.com/kevinmgranger/pelorus.git`, `source_ref: log_version_at_startup`
2. Look at the top of the logs for a pod from that deployment
3. See the logged repo, branch / ref, and commit information
